terraform/environment/region/modules/lambda/variables.tf

```diff
@@ -59,3 +59,8 @@ variable "ecr_arn" {
 variable "kms_key" {
   description = "KMS key for the lambda log group"
 }
+
+variable "iam_policy_documents" {
+  type    = list(string)
```

Comment:
because there's behaviour here, it would be worth adding a description, like "List of IAM policy documents that are merged together. Documents earlier in the list override later ones"